app/views/sephora_style_guide/home/index.slim

```diff
@@ -53,9 +77,25 @@ javascript:
       isOutOfStock: false,
       showAlertSuccess: false,
       showAlertWarning: false,
-      showAlertError: false
+      showAlertError: false,
+      showSidebar: false,
+      elements: [ { name: 'Colors', id: '#colors' }, { name: 'Typography', id: '#typography'}, { name: 'Buttons', id: '#buttons' }, { name: 'Input Fields', id: '#forms' }, { name: 'Tables', id: '#tables' }, { name: 'Grid', id: '#grid' } ],
```

Comment:
Would be better indented. And you can consider declaring it as a var slightly up above for readability?

Reply:
Is what I've done alright? I indented it and declared it as a variable before the vue declaration.

Reply:
Yeah better. Though having one on each line I think will be easier to follow:
```js
var elements = [
  { name: 'Colors', id: '#colors' },
  { name: 'Typography', id: '#typography'},
  // ...
]
```